python/snewpy/models/ccsn.py

```diff
@@ -742,6 +742,51 @@ def __new__(cls, filename=None, *, progenitor_mass=None):
     # Populate Docstring with abbreviated param values
     __new__.__doc__ = __new__.__doc__.format(**_param_abbrv)
 
+
+class Mori_2023(_RegistryModel):
+    """Model based on 2D simulations with axionlike particles, K. Mori, T.  Takiwaki, K. Kotake and S. Horiuchi, Phys. Rev. D 108:063027, 2023. All models are based on the non-rotating 20 M_sun solar metallicity progenitor model from S.E. Woolsey and A. Heger, Phys. Rep. 442:269, 2007. Data from private communication.
+        """
+    param = {'axion_mass' : [0, 100, 200],
+             'axion_coupling' : [0, 2, 4, 6, 8, 10, 12, 14, 16, 20] }
```

Comment:
I think it would be better for the user to operate the dimensional input parameters, i.e. `u.Quantity` (the same we do with `progenitor_mass` in other models, for example), and not work numbers in our internal units.

Maybe we should make the input parameters dimensional here
```suggestion
    param = {'axion_mass' : [0, 100, 200]<<u.MeV,
             'axion_coupling' : [0, 2, 4, 6, 8, 10, 12, 14, 16, 20]<<(1e-10/u.GeV) }
```